engine/src/main/kotlin/world/gregs/voidps/engine/data/sql/DatabaseStorage.kt

```diff
@@ -48,10 +48,10 @@ class DatabaseStorage : AccountStorage {
             .groupBy(AccountsTable.id)
             .associate { row ->
                 val variables: Map<String, Any> = row[names].mapIndexed { index, s -> s to if (s == "coin_share_setting") row[booleans][index] else row[strings][index] }.toMap()
-                val playerName = row[AccountsTable.name]
```

Comment:
Lowercase account stuff looks good, it would be nice to apply it to FileStorage too so behaviour is consistent even across OS's